tests/foreman/conftest.py

```diff
@@ -34,6 +34,7 @@
 except ImportError:
     pass
 from robottelo.config import settings
+from robottelo.constants import DEFECT_TYPES
```

Comment:
why this constant is needed here?

Reply:
Yeah, gotta remove it ! I imported it during testing this PR.